components/hidden_data_box/wikis/starcraft2/hidden_data_box_custom.lua

```diff
@@ -7,32 +7,39 @@
 --
 
 local Class = require('Module:Class')
+local Logic = require('Module:Logic')
 local Variables = require('Module:Variables')
+
 local BasicHiddenDataBox = require('Module:HiddenDataBox')
 local CustomHiddenDataBox = {}
 
 function CustomHiddenDataBox.run(args)
 	BasicHiddenDataBox.addCustomVariables = CustomHiddenDataBox.addCustomVariables
+	if args.team_number or Logic.readBool(args.teamevent) then
+		args.participantGrabber = true
+	else
+		args.participantGrabber = false
```

Comment:
will not work due to the `or true` in the main module
either set `'false'` (string not bool) here or use Logic.emptyOr in the main module